glslc/src/dependency_info.h

```diff
@@ -0,0 +1,163 @@
+// Copyright 2015 The Shaderc Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GLSLC_DEPENDENCY_INFO_H
+#define GLSLC_DEPENDENCY_INFO_H
+
+#include <set>
+#include <string>
+#include <string>
+
+namespace glslc {
+class DependencyInfoDumpingHandler {
+ public:
+  // Sets the dependency target explicitly. It's the same as the argument to
+  // -MT.
+  void SetDepTarget(const std::string& target_label) {
+    user_specified_dep_target_label_ = std::string(target_label);
+  }
+
+  // Sets the name of the file where dependency info will be written.
+  void SetDepFileName(const std::string& dep_file_name) {
```

Comment:
Be consistent between "Dep" and "Dependency" in your methods.
SetDependencyFileName